src/components/GemsPage.js

```diff
@@ -44,17 +37,13 @@ class GemsPage extends Component {
     this.setState({
       searchedGems
     })
-    console.log('searched gems',searchedGems)
   }
 
+
   render() {
     const GemPageDisplay = this.state.GemPage ? { display: 'none' } : {};
     const gemCards = this.state.searchedGems.map(gem => {
-      return <GemCard key={gem.id} gemName={gem.name} gemFam={gem.family} gemImg={gem.image} />
-    })
-
-    const metalsCards = this.props.metals.map(metal => {
-      return <MetalCard key={metal.id} metalName={metal.name} metalColors={metal.color} metalMakeup={metal.makeup} durable={this.determineDurability}/>
+      return <GemCard selectGem={this.props.selectGem} key={gem.id} gemId={gem.id} gemName={gem.name} gemFam={gem.family} gemImg={gem.image} />
```

Comment:
Let's give this component a little room to breathe, aye? Might be a little more readable if we put each prop on its own line.